tests/fixtures/cdn2.thecatapi.com.images-cjd.jpg.php

```diff
@@ -0,0 +1,31 @@
+<?php
+declare(strict_types = 1);
+
+return [
+    'authorName' => null,
+    'authorUrl' => null,
+    'cms' => null,
+    'code' => [
+        'html' => '<img src="https://cdn2.thecatapi.com/images/cjd.jpg"></img>',
```

Comment:
The `img` shouldn't have a closing tag

Reply:
Ah, yes, you are right. I see the following options:
- adding an special case for `<img>` to the `html()` function (my favorite)
- adding a `$selfClosing` parameter to the `html()` function
- replacing `></img>` in the generated html with ` />` (yeah, just kidding :smile:)
- generating the img tag directly without the `html()` function

What would you prefer?

Reply:
Yep, agree. I like the first option (the special case)